__test__/eeris_averages.test.js

```diff
@@ -0,0 +1,145 @@
+const got = require("got").extend({ baseUrl: process.env.CENOTE_API_URL, json: true, throwHttpErrors: false });
+
+const { PROJECT_ID, CENOTE_MASTER_KEY } = process.env;
+const { NUM_OF_DOCS } = global;
+const installationId = "5d80e4c5e209594310f3bb01";
```

Comment:
Maybe make this an env variable as well? I already added it as `EERIS_INSTALLATION_ID` to Travis

Reply:
Proceeded with the requested change.